app/src/main/java/com/worldsnas/starplayer/view/PlayerFragment.kt

```diff
@@ -69,4 +65,16 @@ class PlayerFragment : Fragment() {
         releaseExoPlayer()
         viewBinding = null
     }
+
+    private fun setMusicArguments() {
+        viewBinding?.tvMusicTitle?.text = arguments?.getString("title")
```

Comment:
ye data class besazi ke in parametr haro negah dare
baad class ro parcelable befrestid be in fragment
injori kheili shirin tare zendegi
bazam jaye tozih dare

Reply:
yani bundle moshkel dare?
ye modle dg besazam?

Reply:
mitonid az "safe args" navigation component estefadeh konid kheili karo rahat mikoneh
inam link khode google

https://developer.android.com/guide/navigation/navigation-pass-data#override_a_destination_argument_in_an_action

Reply:
na bundle moshkeli nadare
farghi ham nemikone az safeargs estefade kardid ya na
ye kelas bepichid dore in parametr haye ersali va on kelas ro pass bedid bejaye khode parameter ha